sdcm/nemesis/monkey/sla.py

```
@@ -0,0 +1,229 @@
+            role.attached_service_level.session = session
+            role.session = session
+            try:
+                self.runner.actions_log.info(f"Dropping service level {role.attached_service_level_name}")
+                role.attached_service_level.drop(if_exists=False)
+                time.sleep(300)  # let load to run without service level for 5 minutes
+            finally:
+                self.runner.actions_log.info(f"Re-creating service level {role.attached_service_level_name}")
+                role.attach_service_level(
+                    ServiceLevel(
+                        session=session,
+                        name=SERVICE_LEVEL_NAME_TEMPLATE % (removed_shares, random.randint(0, 10)),
```

Ruff (0.15.15)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)